### PR TITLE
Fix/mutoboteam 2030 pixel glitch mutoboteam 1976 a tag seo

### PR DESCRIPTION
**Pull request by Weedshaker**

### CODE REVIEW

**Review by Weedshaker — COMMENTED** [review]
linted, tested and reviewed

File: src/es/components/controllers/recipe/Recipe.js [hunk 1]
```diff
@@ -200,4 +238,22 @@ export default class Recipe extends Shadow() {
...
+      if (addToTitle) {
+        document.title = document.title.replace(new RegExp(`(.*)${addToTitle.replace(/\s/g, '\\s').replace(/\|/g, '\\|')}.*`), '$1')
+        document.title += addToTitle + textContent
+      } else if (document.title.includes('|')) {
+        document.title = document.title.replace(/[^|]*(.*)/, textContent + ' $1')
+      } else {
+        document.title = textContent
+      }
+    }
+  }
```
Comment (ASchmidt1024) [inline src/es/components/controllers/recipe/Recipe.js:258]: (code improvement proposal for `setTitle()` see above)

File: src/es/components/msrc/productList/ProductList.js [hunk 2]
```diff
@@ -213,4 +214,22 @@ export default class ProductList extends Intersection(Prototype()) {
...
+      if (addToTitle) {
+        document.title = document.title.replace(new RegExp(`(.*)${addToTitle.replace(/\s/g, '\\s').replace(/\|/g, '\\|')}.*`), '$1')
+        document.title += addToTitle + textContent
+      } else if (document.title.includes('|')) {
+        document.title = document.title.replace(/[^|]*(.*)/, textContent + ' $1')
+      } else {
+        document.title = textContent
+      }
+    }
+  }
```
Comment (ASchmidt1024) [inline src/es/components/msrc/productList/ProductList.js:234]: (code improvement proposal for `setTitle()` see above)

**Review by ASchmidt1024 — APPROVED** [review]
Very nice! 🚀

**Pull request merged**